app/src/components/common/TmPage.vue

```diff
@@ -109,19 +51,11 @@ export default {
     ...mapState([`session`]),
     ...mapGetters([`connected`]),
   },
-  watch: {
-    $route() {
-      this.scrollContainer.scrollTop = 0
```

Comment:
my understanding is that this isn't actually necessary / doing anything because of https://github.com/luniehq/lunie/pull/4488/files#diff-e2d8a102050232d3ad7188cabd6720b5R64

but i could be wrong

Reply:
that code looks cool. I didn't even know you could do that with Vue. Let me check